catalog-info.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: "public-cloud-techdocs"
+  title: "Public cloud development guide"
   description: "A guide to using Public Cloud in the Government of the Province of British Columbia"
```

Comment:
description: "Learn about building and deploying applications through B.C. government AWS landing zone."

Reply:
added - over to you @MaxWardle  :)